analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RoslynSymbolicExecution.cs

```diff
@@ -131,6 +131,13 @@ private ExplodedNode ProcessBranch(ExplodedNode node, ControlFlowBranch branch)
         {
             return FromFinally(node.FinallyPoint.CreateNext());     // Redirect from finally back to the original place (or outer finally on the same branch)
         }
+        else if (branch.Semantics == ControlFlowBranchSemantics.Rethrow
+            && branch.Source.BranchValue is { } branchValue)
```

Comment:
```suggestion
            && branch.Source.BranchValue is { } branchValue) // Only for conditional branching
```

Reply:
I don't se the `// Only for conditional branching` comment

Reply:
Or the main comment on line 136 does explain it enough?

Reply:
That was my intent, yes. Would it be better to move the comment one line up?

Reply:
Good enough as is